CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [v1.12.0](https://github.com/openebs/api/tree/v1.12.0) (2020-07-15)
```

Comment:
Why didn't we created 1.12.0 tag?

Reply:
We generally create the api tag once the release is done. So i just created